### DISCUSSION TITLE
Error in solve

### DISCUSSION THREAD

**Discussion by GeekWizkid** [DanWBR/dwsim#592] [Q&A]
why do you count the enthalpy of the reaction at a temperature of 25 c in PFR and not at the current temperature? 
```For Each sb As Compound In ims.Phases(0).Compounds.Values
                            If N.ContainsKey(sb.Name) Then
                                DHr += sb.ConstantProperties.IG_Enthalpy_of_Formation_25C * sb.ConstantProperties.Molar_Weight * (N(sb.Name) - N0(sb.Name)) / 1000
                            End If
                        Next```

**Comment by DanWBR** [comment]
Please read this book: https://www.amazon.com/Introduction-Chemical-Engineering-Thermodynamics-International-dp-1260597687/dp/1260597687/ref=dp_ob_title_bk
